inc/class-sitemaps-categories.php

```diff
@@ -43,7 +44,7 @@ class Core_Sitemaps_Categories extends Core_Sitemaps_Provider {
 	 * @param int    $page_num Page of results.
 	 * @return array $url_list List of URLs for a sitemap.
 	 */
-	public function get_url_list( $object_type, $page_num = 1 ) {
+	public function get_url_list( $page_num = 1 ) {
```

Comment:
Removing this is fine but you'll need to update the filter below, which is still referencing this variable. For now, you can pass "category" to the filter instead of `$object_type` and we'll circle back and update this as a part of #22.